Specs/DateTime/German/DateTimeModel.json

```diff
@@ -972,8 +991,8 @@
     "Context": {
       "ReferenceDateTime": "2016-11-07T16:12:00"
     },
-    "Comment" : "Output contains a trailing whitespace",
-    "NotSupported": "dotnet, javascript, python",
+    "Comment" : "Output contains a trailing whitespace, morgens in this context means morning but is getting recognized as morgen = tomorrow",
```

Comment:
How would you write in German "I'll be away from 5 to 7 in the morning.", "I'll be away from 5 to 7 tomorrow.", and "I'll be away  tomorrow from 5 to 7."?

Reply:
"I'll be away from 5 to 7 in the morning." -> "Ich werde von 5 bis 7 am morgen nicht da sein." or "Ich werde von 5 bis 7 morgens nicht da sein"
 "I'll be away from 5 to 7 tomorrow." -> "Ich werde morgen von 5 bis 7 nicht da sein" or "Ich werde von 5 bis 7 morgen nicht da sein" or "Morgen werde ich von 5 bis 7 nicht da sein"
"I'll be away tomorrow from 5 to 7." -> see above. This position of "tomorrow" wouldn't work in German.